src/pages/_app.tsx

```diff
@@ -25,7 +25,6 @@ import {
 } from '@hashicorp/platform-analytics'
 import '@hashicorp/platform-util/nprogress/style.css'
 import useAnchorLinkAnalytics from '@hashicorp/platform-util/anchor-link-analytics'
-import CodeTabsProvider from '@hashicorp/react-code-block/provider'
```

Comment:
I removed this provider since the corresponding `CodeBlock` and `CodeTabs` from the same package are no longer being used.